### PR TITLE
Prettier

### PR DESCRIPTION
**Pull request by pyr0gan**
- Configuration added to .prettierrc.yaml

### CODE REVIEW

File: .prettierrc.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (guillaumegaluz) [inline .prettierrc.yaml:1]: remove empty line

File: package.json [hunk 2]
```diff
@@ -12,7 +12,13 @@
...
     "near:test:verbose": "yarn near:build:contract:debug && cd contracts/near && cargo test -v -- --nocapture && cd -",
-    "truffle-polygon": "yarn global add truffle && cd ./contracts/polygon/SimpleStorage && yarn && truffle obtain --solc 0.8.0"
+    "truffle-polygon": "yarn global add truffle && cd ./contracts/polygon/SimpleStorage && yarn && truffle obtain --solc 0.8.0",
+    "secret:init": "curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh && source $HOME/.cargo/env && yarn secret:tools",
+    "secret:tools": "rustup default stable && rustup target list --installed && rustup target add wasm32-unknown-unknown && rustup install nightly && rustup target add wasm32-unknown-unknown --toolchain nightly && yarn secret:cargo-generate",
+    "secret:cargo-generate": "cargo install cargo-generate --features vendored-openssl && yarn secret:contract",
+    "secret:contract": "cd contracts/secret && cargo generate --git https://github.com/enigmampc/secret-template --name mysimplecounter",
+    "secret:test": "cd contracts/secret/mysimplecounter && cargo test",
+    "secret:compile": "cd contracts/secret/mysimplecounter && cargo wasm && cargo schema"
```
Comment (guillaumegaluz) [inline package.json:21]: The PR is named 'Prettier' but adding those scripts is nto related to prettier. 

Did you mean to add them?

File: pages/api/secret/transfer.ts [hunk 3]
```diff
@@ -36,7 +37,7 @@ export default async function connect(
         const rcpt = address; // Set recipient to sender for testing, or generate another account as you did previously.
         const memo = 'sendTokens example'; // optional memo
         const sent = await client.sendTokens(rcpt, [{ 
-          amount: '1000000', 
+          amount: txAmount,
```
Comment (guillaumegaluz) [inline pages/api/secret/transfer.ts:40]: is this related to prettier?

**Comment by guillaumegaluz** [comment]
I think you made this PR off the same branch as the Secret PR because some of the code is in common. we can fix another time but as per-Slack: let's put this on pause for now 👍🏼

**Comment by zurgl** [comment]
Later If this feature is still a requirement I will be do it. Then when can close this PR for now
